config/settings.yml

```
@@ -8,3 +8,6 @@ course:
     name_max_255: 255
     max_1000: 1000
   num_demo_4: 4
+pagy:
+  default_page: 1
+  max_items_per_page: 16
```

```suggestion
  default_page_1: 1
  per_page_16: 16
```


ri để đọc tên biến là hiểu lun em nhé

---

da vang a